src/rust/ide/view/src/documentation.rs

```diff
@@ -255,6 +260,8 @@ ensogl::define_endpoints! {
         display_documentation (String),
         /// Display documentation represented by docstring.
         display_docstring (String),
+        /// Display documentation represented by Html code.
+        display_doc_html (String),
```

Comment:
Please remove anything related to old documentation, if it isn't needed anymore.